### ISSUE TITLE
What is the best practice for publishers to provide contact information for reporting errors and other issues. 

### ISSUE THREAD

**Issue by GeorgeKerscher** [w3c/publ-a11y#98]
The accessibilitySummary is potentially a place for publishers to provide information about reporting errors and other issues. Perhaps this could follow the conformsTo field? It might not belong in the accessibilitySummary at all. However, this best practice needs to be brought up.

**Comment by clapierre** [comment]
I think it should be included here.  Say this title only meets WCAG-A, they could state in the AccessibilitySummary that the only WCAG-AA failure is that images containing text do not meet WCAG-AA color contrast requirements due to copyright and keeping with the print equivalent book.

**Comment by GeorgeKerscher** [comment]
On the May 19 call of the sub-taskforce, we decided that it was best not to include contact information for the publisher in the accessibility summary. There were many reasons, including that an issue that is really a reading system problem may end up with the publisher. The publishers put contact information on their website and perhaps in the book. It would also be difficult to have links in metadata fields, which show up as free text.
